### DISCUSSION TITLE
GitHub Codespace Error 'Failed to fetch'

### DISCUSSION THREAD

**Discussion by Connorado07** [community/community#40885] [Codespaces]
### Select Topic Area

Bug

### Body

So, I was opening up my Codespace in Firefox yesterday when i kept getting a connection error reading "NetworkError when attempting to fetch resource." After researching it, it led be down a rabbit hole that basically said to use Google Chrome for it. So I did. I moved over to chrome and I now 
get the error of 'Failed to fetch'. No idea at this point. Here is what it looks like:
![GitHubCodespacesError1](https://user-images.githubusercontent.com/77865401/205815983-7cfc02c1-f0c4-469a-a1a6-2ee568e224ec.png)

And here is my chrome console during this process of failing to load:
![GitHubCodespacesError](https://user-images.githubusercontent.com/77865401/205816040-3b0382b8-3810-4b8e-84a6-1d57a79cc1d7.png)

Thought that I would report this bug and maybe someone has a solution.
Thanks, Connorado.

**Comment by ossanna16** [comment]
Hi there @ronnoxtyb and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by eljog** [comment]
Hi @ronnoxtyb 
Upon investigation it appears that your codespace was affected by an [incident](https://www.githubstatus.com/incidents/by950fpm0xlg) in Codespaces service. We have had the issue resolved on that day and you should be able to access your codespace  fine.
Apologies for the inconvenience caused, and thank you for reaching out to us.

**Reply by nihalsheikh** [reply]
Did anyone find the solution, i am also encountering the same issue. is this error server related, beacuse i found another thread [https://github.com/orgs/community/discussions/122950](https://github.com/orgs/community/discussions/122950) with same issue for Southeast Asia server

**Comment by JPro561** [comment]
You can open your codespace directly in Visual Studio Code by clicking the button next to it. Just make sure you have the GitHub Codespaces extension installed. This worked for me, so it might work for you too.
